newIDE/app/src/UI/HelpIcon/index.js

```diff
@@ -9,7 +9,7 @@ type PropsType = {|
   helpPagePath: ?string,
   disabled?: boolean,
   style?: {|
-    padding?: number,
```

Comment:
are you using the helpicon?

Reply:
No but style props must match since HelpIcon uses IconButton and Flow wants the `style` props to be identical